### PR TITLE
Add July 2023 coworking

### PR DESCRIPTION
**Pull request by steffilazerte**

### CODE REVIEW

**Comment by steffilazerte** [comment]
Hi @maelle,
When you get a chance can you take a [look](https://deploy-preview-562--ropensci.netlify.app/events/coworking-2023-07/) at the preview event and let me know if you haven any suggestions for wording, etc.?
No hurry 😁, thanks!

File: content/events/2023-07-04-coworking-2023-07.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: Social Coworking and Office Hours - Create/Update your 'Happy' File!
```
Comment (maelle) [inline content/events/2023-07-04-coworking-2023-07.md:2]:
```suggestion
title: Social Coworking and Office Hours - Create/Update your 'Happy File'/'Brag Document'!
```

File: content/events/2023-07-04-coworking-2023-07.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+d <- lubridate::ymd_hms('2023-07-04 14:00:00', tz = 'Europe/Paris')
+lubridate::with_tz(d, 'UTC')
+lubridate::with_tz(d, 'America/Winnipeg')
+```
+-->
+
+**Join us for 2 hours Tuesday July 04, 14:00 European Central (12:00 UTC) for 
+[Social Coworking + Office Hours](/blog/2021/08/17/coworking-sessions/)**
+
+This month we're focusing on the theme **Create/Update your 'Happy' File!**
```
Comment (maelle) [inline content/events/2023-07-04-coworking-2023-07.md:36]:
```suggestion
This month we're focusing on the theme **Create/Update your 'Happy File' or ['Brag Document'](https://jvns.ca/blog/brag-documents/), listing your small and big accomplishments at work!** 
```

File: content/events/2023-07-04-coworking-2023-07.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+[Social Coworking + Office Hours](/blog/2021/08/17/coworking-sessions/)**
+
+This month we're focusing on the theme **Create/Update your 'Happy' File!** 
+with co-host **Maëlle Salmon**, rOpenSci's own Research Software Engineer!
+
+Come for the full 2 hrs or only as long as you need!
+
+### Cowork
+
+- Explore what goes into a 'Happy File' and why you need one
```
Comment (maelle) [inline content/events/2023-07-04-coworking-2023-07.md:43]:
```suggestion
- Explore what goes into a 'Happy File'/'Brag Document' and why you need one
```

**Review by maelle — CHANGES_REQUESTED** [review]
Thank you!! I added "brag document" everywhere as it might be more known.
I wonder whether it'd be weird to link to https://twitter.com/JennyBryan/status/1582862196870373377 -- that's where I learnt the phrase and know no other source.

File: content/events/2023-07-04-coworking-2023-07.md (hunk 2 above)
Comment (steffilazerte) [reply content/events/2023-07-04-coworking-2023-07.md:36]: I think that linking "Happy File" to Jenny's post here would be fine, it's a public source and gives some context/credit to where you got the term.

**Pull request merged**